tests/2020-09-01-fastcore.ipynb

```diff
@@ -1,14 +1,5 @@
 {
```

Comment:
<p>I had an overcomplicated test where I was trying to merge raw front matter with markdown front matter.  Currently the way the front matter processor works is it doesn't try to update the raw front matter if it detects a <code>title</code> field.  I think this is perfectly reasonable, and I changed this test notebook accordingly</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/fastai/nbdev/pull/1002/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>